src/styles/blocks/header.scss

```diff
@@ -0,0 +1,76 @@
+.header {
+  background-color: $background-color-special-white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__nav {
+    font-size: 12px;
+    font-weight: 500;
+
+    overflow-x: auto;
```

Comment:
The `overflow-x: auto` property on the navigation bar might cause a horizontal scrollbar to appear if the content overflows. Ensure this behavior is intended and visually acceptable.